docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -645,3 +645,48 @@ The service will respond with the two identity IDs created:
   ]
 }
 ```
+
+### Errors during bulk import
+
+Failure to import an identity will not fail the whole bulk import. In the response, `"action": "error"` indicates thath the
+identity with the corresponding `patch_id` could not be imported. The `error` object contains more details about why the import
```

Comment:
What is patch_id?

Reply:
> Optionally, you can specify a `patch_id` property which will be returned in the response. This can be used to
correlate the response with the patch.